egs/voxceleb/v1/local/make_voxceleb1_v2.pl

```diff
@@ -40,11 +40,11 @@
     my @rec_dirs = grep {-d "$data_base/$dataset/wav/$spkr_id/$_" && ! /^\.{1,2}$/} readdir($dh);
     closedir $dh;
     foreach (@rec_dirs) {
-	  my $rec_id = $_;
-	  opendir my $dh, "$data_base/$dataset/wav/$spkr_id/$rec_id/" or die "Cannot open directory: $!";
-	  my @files = map{s/\.[^.]+$//;$_}grep {/\.wav$/} readdir($dh);
-	  closedir $dh;
-  	  foreach (@files) {
+      my $rec_id = $_;
```

Comment:
this diff is hard to see because of all the whitespace changes.   Any chance of fixing that?

Reply:
actually its OK, I'll merge as-is.

Reply:
I believe that only using whitespace for indentation is a better code style. I am sorry for that I didn't explain it earlier for those whitespace changes. Anyway, you finally got it.